Dangerfile

```diff
@@ -56,14 +64,55 @@ modified_files.each do |file|
       # Make sure resolves merges or rebases conflict issues
       fail("Commited file without resolving merges/rebases conflict issues on `#{file}` - `#{line}`") if line =~ /^>>>>>>>/
       # Look for Amazon Secret keys in modified files
-      warn("Possible amazon secret key hardcoded found in `#{file}`") if line =~ /(?<![A-Za-z0-9\/+=])[A-Za-z0-9\/+=]{40}(?![A-Za-z0-9\/+=])/
+      warn("Possible amazon secret key hardcoded found in `#{file}`") if line =~ /(?<![A-Za-z0-9\/+=])[A-Za-z0-9\/+=]{40}(?![A-Za-z0-9\/+=])/ && file != "yarn.lock"
     end
     rescue
       message "Could not read file #{file}, does it really exist?"
     end
 end
 
 ########################
+#   FUNCTIONS SECTION  #
+########################
+
+def checkForNodeVersion(file, line)
+  # Keep node version synced between declarations
+  warn("`package.json` node version was modified. Remember to update node version on `.travis.yml`, `.nvmrc` and `README`!") if file && line && file == "package.json" && line =~ /"node":/
+  warn("`.nvmrc` was modified. Remember to update node version on `.travis.yml`, `package.json` and `README`!") if file && file == ".nvmrc"
+end
+
+def checkForWeaklyTypedFunctionReturn(line)
+  # Warn when a TypeScript file has a new function returning <any> instead of strongly typed.
+  # There are several situation that need to return just 'any', so to avoid having too many false positives 
+  #   we are checking just <any> for now
+  warn("Possibly returning <any> in a function, prefer having a strongly typed return. `#{file}` at line `#{line}`.") if line =~ /<any>/im
```

Comment:
can you also change to `Possibly returning generic any in a function...` due to #15